decam_process.py

```diff
@@ -15,324 +15,378 @@
+
+    Parameters
+    ----------
+    repo: `str`
+        The output repository location on disk where ingested raw images live.
+    calibrepo: `str`
+        The output repository location on disk where ingested calibration images live.
+    calibdatafiles: `list`
+        A list of the filenames of each flat and bias image file.
+    defectfiles: `list`
+            A list of the filenames of each defect image file.
+            The first element in this list must be the name of a .tar.gz file
+            which contains all the compressed defect images.
 
     BASH EQUIVALENT:
-    $ ingestCalibs.py repo --calib calibrepo --mode=link --validity 999 datafiles
+    $ ingestCalibs.py repo --calib calibrepo --mode=link --validity 999 calibdatafiles
+    $ cd calibrepo
+    $ ingestCalibs.py ../../repo --calib . --mode=skip --calibType defect --validity 999 defectfiles
+    $ cd ..
 
     RESULT:
-    calibrepo populated with *links* to files in datadir,
-    organized by date (bias/zero and flat images only)
-    sqlite3 database registry of ingested calibration products
-    created in calibrepo
+    calibrepo populated with *links* to calibdatafiles,
+    organized by date (bias and flat images only)
+    sqlite3 database registry of ingested calibration products (bias, flat,
+    and defect images) created in calibrepo
+
+    NOTE:
+    calib ingestion ingests *all* the calibs, not just the ones needed
+    for the specified visits. We may want to revisit this in the future.
     '''
+
+    def flatBiasIngest(repo, calibrepo, calibdatafiles):
+        '''
+        Ingest DECam flats and biases
+    
+        Parameters
+        ----------
+        repo: `str`
+            The output repository location on disk where ingested raw images live.
+        calibrepo: `str`
+            The output repository location on disk where ingested calibration images live.
+        calibdatafiles: `list`
+            A list of the filenames of each flat and bias image file.
+    
+        '''
+        print('Ingesting flats and biases...')
+        args = [repo, '--calib', calibrepo, '--mode', 'link', '--validity', '999']
+        args.extend(calibdatafiles)
+        argumentParser = IngestCalibsArgumentParser(name='ingestCalibs')
+        config = IngestCalibsConfig()
+        config.parse.retarget(ingestCalibs.DecamCalibsParseTask)
+        ingestTask = IngestCalibsTask(config=config, name='ingestCalibs')
+        parsedCmd = argumentParser.parse_args(config=config, args=args)
+        try:
+            ingestTask.run(parsedCmd)
+        except sqlite3.IntegrityError as detail:
+            print('sqlite3.IntegrityError: ', detail)
+            print('(sqlite3 doesn\'t think all the calibration files are unique)')
+        else:
+            print('Success!')
+            print('Calibrations corresponding to {0} are now ingested in {1}'.format(repo, calibrepo))
+        return
+
+    def defectIngest(repo, calibrepo, defectfiles):
```

Comment:
I was wondering when you'd say that. I realized the same thing and have started writing some!